com_biblestudy/admin/language/de-DE/de-DE.com_biblestudy.ini

```diff
@@ -302,45 +302,7 @@ JBS_ADM_FULL_WINDOW="Vollbild"
 JBS_ADM_HITS_PLAYS_DOWNLOAD="Treffer + abgespielt + Downloads"
 JBS_ADM_INTERNAL_SHARE="BibleStudy interne Social Network Funktion"
 JBS_ADM_JWPLAYER="JW Player"
-JBS_ADM_JWPLAYER_ADOBE_SITECATALYST="Adobe SiteCatalyst"
-JBS_ADM_JWPLAYER_ADOBE_SITECATALYST_DESC="Configures the Adobe SiteCatalyst connection for the Premium and Ads editions."
-JBS_ADM_JWPLAYER_ADVERTISING="Advertising"
-JBS_ADM_JWPLAYER_ADVERTISING_DESC="Configures the VAST or IMA Advertising options for the Ads edition."
-JBS_ADM_JWPLAYER_AUTOSTART_DESC="Automatically start playing the video on page load. Can be true or false (default). Autostart does not work on mobile devices like iOS and Android."
-JBS_ADM_JWPLAYER_CAPTIONS="Captions"
-JBS_ADM_JWPLAYER_CAPTIONS_DESC="Configures the layout of the closed captions rendering area (load captions using a playlist)."
-JBS_ADM_JWPLAYER_CDN_LABEL="JWPLayer CDN URL"
-JBS_ADM_JWPLAYER_CONTROL="Controls"    ; translate
-JBS_ADM_JWPLAYER_FALLBACK="FallBack"
-JBS_ADM_JWPLAYER_FALLBACK_DESC="Whether to render a nice download link for the video if HTML5 and/or Flash are not supported. Can be true (a fallback is rendered) or false (the original HTML is not touched). Defaults to true."
-JBS_ADM_JWPLAYER_GOOGLE_ANALYTICS="Google Analytics"
-JBS_ADM_JWPLAYER_GOOGLE_ANALYTICS_DESC="Configures the Google Analytics connection for the Premium and Ads editions."
-JBS_ADM_JWPLAYER_IMAGE="Image"
-JBS_ADM_JWPLAYER_IMAGE_DESC="URL to a JPG/PNG poster image to display before playback is started. For audio-only media, the poster image stays visible during playback."
 JBS_ADM_JWPLAYER_KEY_LABEL="Player Lizenzschlüssel (auf dem eigenen Server)"
-JBS_ADM_JWPLAYER_LISTBAR="ListBar"
```

Comment:
Yes. Currently it takes mee too much time to translate all this new phrases to german language. So I remove them in german language only. The engish phrase is automatically replacing the german / not english one, if a key is missing in germain / not english version. (Also it is easier to find a missing translation).